CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - Move Bibtex and Biblatex mode switcher to File menu
 - Display active edit mode (BibTeX or Biblatex) at window title
 - Implements #444: The search is cleared by either clicking the clear-button or by pressing ESC with having focus in the search field. 
+- IEEEXplore search now downloads the full Bibtex record instead of parsing the fields from the HTML webpage result (fixes #1146, #1267)
```

Comment:
Not sure if this is a valid comment, but it looks like you wrote `#1146` etc, which I think should be `feature 1146` (see top of the file).